services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -88,44 +133,102 @@ const Asset = ({ libraryData }) => {
             aspectRatio={{ sm: '2x1', md: '1x1', lg: '3x4', xlg: '1x1' }}
             border={['sm']}
           >
-            Maintainer
+            <Column className={dashboardStyles.title}>Maintainer</Column>
+            <Column className={dashboardStyles.maintainer}>{libraryData.content.name}</Column>
+            <Column className={dashboardStyles.title}>
+              <AssetItemMeta className={dashboardStyles.meta} properties={['status']} />
+            </Column>
+            <Column>
+              {SponsorIcon && <SponsorIcon className={dashboardStyles.metaAbsolute} size={64} />}
+            </Column>
           </DashboardItem>
         </Column>
         <Column className={dashboardStyles.column} lg={2}>
           <DashboardItem aspectRatio={{ sm: '1x1', lg: 'none', xlg: 'none' }} border={['sm']}>
             <Grid columns={2} className={dashboardStyles.subgrid}>
-              <Column className={dashboardStyles.subcolumn}>Library</Column>
-              <Column className={dashboardStyles.subcolumn}>Version</Column>
-              <Column className={dashboardStyles.subcolumn}>License</Column>
-              <Column className={dashboardStyles.subcolumn}>Framework</Column>
-              <Column className={dashboardStyles.subcolumn}>Last modified</Column>
-              <Column className={dashboardStyles.subcolumn}>Design kit</Column>
+              <Column className={clsx(dashboardStyles.library, dashboardStyles.subcolumn)}>
+                <Column className={dashboardStyles.title}>Library</Column>
+                <Column>
+                  <Link
+                    href={`/assets/${libraryData.libraryId}/${libraryData.libraryRef || 'latest'}/${
+                      libraryData.assetId
+                    }`}
+                  >
+                    <a className={dashboardStyles.libraryTitle}>{libraryData.content.name}</a>
+                  </Link>
+                </Column>
+              </Column>
+              <Column className={clsx(dashboardStyles.title, dashboardStyles.subcolumn)}>
```

Comment:
I'm seeing a bunch of nested `Column`s that won't do anything as the only child to a parent that isn't a `Grid`.

I'd expect the column to only control layout, and then what's inside the column to control text presentation.

```jsx
<Column className={dashboardStyles.subcolumn}>
  <p className={dashboardStyles.heading}>Version</p>
  <h3 className={dashboardStyles.value}>{libraryData.content.version}</h3>
</Column>
```